lib/data.php

```diff
@@ -53,16 +33,18 @@ function reportExists($reportId, $reportsArray) {
 $reportConfigInstance = new ReportConfig($project_id, $module);
 $config = $reportConfigInstance->getReportConfig();
 $dataDictionary = new DataDictionary(\REDCap::getDataDictionary('array'));
+$db = new Database($rc_connection /* $rc_connection is globabl */);
```

Comment:
The comment has an extra letter in 'global'.

Reply:
Good catch, the others were c/p.